ms2/src/org/labkey/ms2/testMascot.jsp

```diff
@@ -18,73 +18,33 @@
 <%@ page import="org.labkey.api.view.HttpView"%>
 <%@ page import="org.labkey.api.view.JspView" %>
 <%@ page import="org.labkey.ms2.MS2Controller" %>
+<%@ page import="org.apache.commons.lang3.StringUtils" %>
 <%@ page extends="org.labkey.api.jsp.JspBase" %>
 <%
     MS2Controller.TestMascotForm form = ((JspView<MS2Controller.TestMascotForm>)HttpView.currentView()).getModelBean();
 
-    if (null != form.getMessage() && !"".equals(form.getMessage()))
+    if (!StringUtils.isBlank(form.getMessage()))
     {
-        boolean errorMessage;
-        errorMessage = (-1 != form.getMessage().indexOf("Test failed."));
-        if (errorMessage)
-            out.print("<font class=\"labkey-error\">");
-        out.print("<b>" + form.getMessage() + "</b><br><br>");
-        if (errorMessage)
-            out.print("</font>");
+        boolean errorMessage = form.getMessage().contains("Test failed.");
+
+        if (errorMessage) { %><span class="labkey-error"><% } %>
+        <strong><%= h(form.getMessage())%></strong>
+        <% if (errorMessage) { %></span><% }
     }
 %>
 
-<table>
-    <tr><td colspan=2><b>Mascot settings tested:</b></td></tr>
-    <tr><td>Server:</td><td><%
-if (!"".equals(form.getMascotServer()))
-{
-    out.print(form.getMascotServer());
-}
-else
-{
-%>&lt;not set&gt;
-<%}
-%></td></tr>
-    <tr><td>User account:</td><td><%
-if (!"".equals(form.getMascotUserAccount()))
-{
-    out.print(form.getMascotUserAccount());
-}
-else
-{
-%>&lt;not set&gt;
-<%}
-%></td></tr>
-    <tr><td>Password:</td><td><%
-if (!"".equals(form.getMascotUserPassword()))
-{
-    out.print(form.getMascotUserPassword());
-}
-else
-{
-%>&lt;not set&gt;
-<%}
-%></td></tr>
-    <tr><td>HTTP Proxy URL:</td><td><%
-if (!"".equals(form.getMascotHTTPProxy()))
-{
-    out.print(form.getMascotHTTPProxy());
-}
-else
-{
-%>&lt;not set&gt;
-<%}
-%></td></tr>
+<div><strong>Mascot settings tested</strong></div>
+<table class="lk-fields-table">
+    <tr><td colspan=2></td></tr>
+    <tr><td class="labkey-form-label">Server</td><td><%= h(StringUtils.isBlank(form.getMascotServer()) ? "<not set>" : form.getMascotServer()) %></td></tr>

```

Comment:
Could use `StringUtils.defaultIfBlank(form.getMascotServer(), "<not set>")` for these